installation/ilmt-track-bamlcns-kubernetes.adoc

```diff
@@ -0,0 +1,37 @@
+= Tracking BAMOE license use in Kubernetes
+include::../partials/attributes.adoc[]
+
+
+== Prerequisites
+Install the License Service by following the https://www.ibm.com/docs/en/cloud-paks/foundational-services/3.23?topic=software-installing-license-service-stand-alone[tutorial].
```

Comment:
Is the only public reference about License Service is CloudPak docs?

A lot of effort has been done to communicate customers that BAMOE is not part of CloudPak, and now this might confuse users.

Reply:
it's what it's :)